padinfo/core/button_info.py

```diff
@@ -92,36 +93,49 @@ def _get_sub_attr_multiplier(self, monster_model):
-        return INFO_STRING.format(monster.monster_id, monster.name_en, info.main_damage, info.sub_damage,
-                                  info.total_damage,
-                                  info.main_damage_with_atk_latent, info.sub_damage_with_atk_latent,
-                                  info.total_damage_with_atk_latent, card_btn_str, team_btn_str)
+    def to_string(self, dgcog, monster, info):
+        card_btn_str = self._get_card_btn_damage(CARD_BUTTONS, dgcog, monster)
+        team_btn_str = self._get_team_btn_damage(TEAM_BUTTONS, dgcog, monster)
+        return INFO_STRING.format(monster.monster_id, monster.name_en, int(round(info.main_damage)), int(round(info.sub_damage)),
+                                  int(round(info.total_damage)),
+                                  int(round(info.main_damage_with_atk_latent)), int(
+                                      round(info.sub_damage_with_atk_latent)),
+                                  int(round(info.total_damage_with_atk_latent)), card_btn_str, team_btn_str)
 
-    def _get_card_btn_damage(self, card_buttons, info, monster):
+    def _get_card_btn_damage(self, card_buttons, dgcog, monster):
         lines = []
         card_buttons.sort(key=lambda x: x.mult)
         for card in card_buttons:
-            lines.append(CARD_BUTTON_FORMAT.format(card.name, card.mult,
-                         (info.main_damage_with_atk_latent * card.mult)))
+            inherit_model = dgcog.get_monster(card.id)
+            max_level = LIMIT_BREAK_LEVEL if monster.limit_mult != 0 else monster.level
+            inherit_max_level = LIMIT_BREAK_LEVEL if inherit_model.limit_mult != 0 else inherit_model.level
+            stat_latents = dgcog.MonsterStatModifierInput(num_atkpp=monster.latent_slots / 2)
+            dmg = int(round(dgcog.monster_stats.stat(monster, 'atk', max_level, stat_latents=stat_latents,
+                      inherited_monster=inherit_model, multiplayer=True, inherited_monster_lvl=inherit_max_level)))
+            lines.append(CARD_BUTTON_FORMAT.format(
+                card.id, card.name, card.mult, (dmg * card.mult)))
         return "\n".join(lines)
 
-    def _get_team_btn_damage(self, team_buttons, info, monster):
+    def _get_team_btn_damage(self, team_buttons, dgcog, monster):
         # TODO: calculate with oncolor assist damage and ATK+ eq (Oversoul)
         lines = []
         team_buttons.sort(key=lambda x: x.mult)
         for card in team_buttons:
             total_dmg = 0
+            inherit_model = dgcog.get_monster(card.id)
+            max_level = LIMIT_BREAK_LEVEL if monster.limit_mult != 0 else monster.level
+            stat_latents = dgcog.MonsterStatModifierInput(num_atkpp=monster.latent_slots / 2)
+            dmg = dgcog.monster_stats.stat(monster, 'atk', max_level, stat_latents=stat_latents,
+                                           inherited_monster=inherit_model, multiplayer=True, inherited_monster_lvl=inherit_max_level)
```

Comment:
this is where I mean, `inherit_max_level` is only defined in `_get_card_btn_damage`